corehq/apps/users/templates/users/mobile_workers.html

```diff
@@ -242,54 +240,29 @@ <h3 class="panel-title">{% trans 'Mobile Workers' %}</h3>
-                        </select>
-                    </div>
-                    <div class="col-sm-4">
-                        <button type="button"
-                                class="btn btn-default pull-right"
-                                ng-click="showDeactivatedUsers = true; filterData();"
-                                ng-show="!showDeactivatedUsers">
+                    <div class="col-sm-6">
+                        <button type="button" class="btn btn-default pull-right"
+                                data-bind="visible: !inactiveOnly(), click: function() { toggleInactiveOnly(true); }">
                             {% trans 'Show Deactivated Mobile Workers' %}
                         </button>
-                        <button type="button"
-                                class="btn btn-default pull-right"
-                                ng-click="showDeactivatedUsers = false; filterData();"
-                                ng-show="showDeactivatedUsers">
+                        <button type="button" class="btn btn-default pull-right"
+                                data-bind="visible: inactiveOnly(), click: function() { toggleInactiveOnly(false); }">
                             {% trans 'Show Active Mobile Workers' %}
                         </button>
-                        <div class="pull-right" style="clear: right;">
-                            <pagination direction-links="true"
-                                        total-items="total"
-                                        items-per-page="limit"
-                                        previous-text="&laquo;"
-                                        next-text="&raquo;"
-                                        max-size="maxSize"
-                                        ng-model="currentPage"
-                                        ng-change="pageChanged()"></pagination>
-                        </div>
                     </div>
                 </div>
-                <div class="alert alert-info"
-                     ng-show="total_records != 0 && paginatedItems.length == 0 && !notLoaded && !hasError">
+                <div class="alert alert-info" data-bind="visible: showNoUsers">
                     {% blocktrans %}
-                    No Mobile Workers were found that matched your query.
+                        No Mobile Workers were found that matched your query.
                     {% endblocktrans %}
                 </div>
-                <div class="alert alert-info"
-                     ng-show="total_records == 0 && paginatedItems.length == 0 && !notLoaded && !hasError">
+                <div class="alert alert-info" data-bind="visible: showProjectHasNoUsers">
                     {% blocktrans %}
                         <strong>There are currently no Mobile Workers in this project.</strong>
                         Please Create a new Mobile Worker above to get started.
```

Comment:
nit, "Create" and "Mobile Workers" should be lower case

Reply:
7143e8b